source/hands_on/trex.rst

```diff
@@ -0,0 +1,90 @@
+TREX
+^^^^^^^
+
+Overview
+---------
+TREX (Trajectory-ranked Reward Extrapolation) was first proposed in `Extrapolating Beyond Suboptimal Demonstrations via Inverse Reinforcement Learning from Observations <https://arxiv.org/abs/1904.06387>`_, which uses ranked demonstrations to inference reward function. Different from previous methods, TREX seeks a reward function that explains the ranking over demonstrations, rather than justifying the demonstrations. Hence, this algorithm can learn good policies from highly suboptimal demostrations. In addition, the predicted reward only depends on observations, rather than actions.
+

```

Comment:
'Hence, this algorithm can learn good policies from highly suboptimal demostrations'. Why does ranking lead to this feature? this 'Hence' might not be very obvious to audiences.

Also, there is a typo in 'demonstrations'